src/Compilers/CSharp/Test/Emit/CodeGen/SwitchTests.cs

```diff
@@ -9125,81 +9098,497 @@ public static string M2(object o)
+        <entry offset=""0xc4"" startLine=""14"" startColumn=""32"" endLine=""14"" endColumn=""38"" />
+        <entry offset=""0xc6"" hidden=""true"" />
+        <entry offset=""0xc8"" startLine=""15"" startColumn=""24"" endLine=""15"" endColumn=""30"" />
+        <entry offset=""0xd1"" startLine=""15"" startColumn=""32"" endLine=""15"" endColumn=""38"" />
+        <entry offset=""0xd3"" startLine=""16"" startColumn=""24"" endLine=""16"" endColumn=""30"" />
+        <entry offset=""0xdf"" startLine=""16"" startColumn=""32"" endLine=""16"" endColumn=""38"" />
+        <entry offset=""0xe1"" hidden=""true"" />
+        <entry offset=""0xe4"" startLine=""17"" startColumn=""24"" endLine=""17"" endColumn=""30"" />
+        <entry offset=""0xed"" startLine=""17"" startColumn=""32"" endLine=""17"" endColumn=""38"" />
+        <entry offset=""0xef"" startLine=""18"" startColumn=""24"" endLine=""18"" endColumn=""30"" />
+        <entry offset=""0xf8"" startLine=""18"" startColumn=""32"" endLine=""18"" endColumn=""38"" />
+        <entry offset=""0xfa"" hidden=""true"" />
+        <entry offset=""0xfd"" startLine=""19"" startColumn=""24"" endLine=""19"" endColumn=""30"" />
+        <entry offset=""0x106"" startLine=""19"" startColumn=""32"" endLine=""19"" endColumn=""38"" />
+        <entry offset=""0x108"" startLine=""20"" startColumn=""24"" endLine=""20"" endColumn=""30"" />
+        <entry offset=""0x111"" startLine=""20"" startColumn=""32"" endLine=""20"" endColumn=""38"" />
+        <entry offset=""0x113"" startLine=""22"" startColumn=""5"" endLine=""22"" endColumn=""6"" />
+      </sequencePoints>
+      <scope startOffset=""0x0"" endOffset=""0x114"">
+        <scope startOffset=""0xa8"" endOffset=""0xb8"">
+          <local name=""i"" il_index=""2"" il_start=""0xa8"" il_end=""0xb8"" attributes=""0"" />
+        </scope>
+        <scope startOffset=""0xc6"" endOffset=""0xd3"">
+          <local name=""i"" il_index=""3"" il_start=""0xc6"" il_end=""0xd3"" attributes=""0"" />
+        </scope>
+        <scope startOffset=""0xe1"" endOffset=""0xef"">
+          <local name=""i"" il_index=""4"" il_start=""0xe1"" il_end=""0xef"" attributes=""0"" />
+        </scope>
+        <scope startOffset=""0xfa"" endOffset=""0x108"">
+          <local name=""i"" il_index=""5"" il_start=""0xfa"" il_end=""0x108"" attributes=""0"" />
+        </scope>
+      </scope>
+    </method>
+    <method containingType=""Program"" name="".cctor"">
+      <customDebugInfo>
+        <forward declaringType=""Program"" methodName=""Main"" />
+      </customDebugInfo>
+      <sequencePoints>
+        <entry offset=""0x0"" startLine=""8"" startColumn=""5"" endLine=""8"" endColumn=""27"" />
+      </sequencePoints>
+    </method>
+  </methods>
+</symbols>");
+        }
+
+        [Fact, WorkItem(19280, "https://github.com/dotnet/roslyn/issues/19280")]
+        public void TestIgnoreDynamicVsObjectAndTupleElementNames_03()
```

Comment:
>TestIgnoreDynamicVsObjectAndTupleElementNames_03 [](start = 20, length = 48)

I am confused, the name of the test says "Ignore", but it looks like the differences aren't ignored.